robolectric/src/test/java/org/robolectric/android/internal/ConscryptSecurityModeTest.java

```diff
@@ -1,20 +1,18 @@
 package org.robolectric.android.internal;
 
 import static com.google.common.truth.Truth.assertThat;
-import static org.robolectric.annotation.SecurityMode.Mode.CONSCRYPT;
 
 import java.security.cert.CertificateException;
 import java.security.cert.CertificateFactory;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.RobolectricTestRunner;
-import org.robolectric.annotation.SecurityMode;
 
 @RunWith(RobolectricTestRunner.class)
 public class ConscryptSecurityModeTest {
 
   @Test
-  @SecurityMode(CONSCRYPT)
+  //  @SecurityMode(CONSCRYPT)
```

Comment:
If you don't need it, you can remove it directly, instead of commenting it.

Reply:
Done 👍